tools/check-wininstaller-translations.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+set -eu
+
+BASE_DIR=src/res/translation/wininstaller/
+BASE_LANG=en
+INSTALLERLNG=installerlng.nsi
+BASE_LANGSTRINGS=$(grep LangString "${BASE_DIR}/${BASE_LANG}.nsi" | cut -d' ' -f2)
+EXIT=0
+for LANGUAGE_FILE in $(ls -1 src/res/translation/wininstaller/??.nsi | grep -vF "${BASE_LANG}.nsi"); do
+		echo
+		echo "* ${LANGUAGE_FILE}"
+		echo -n "  - Checking language file is included in ${INSTALLERLNG}... "
+		if grep -qF $(tr '/' '\\' <<<"${LANGUAGE_FILE}") "${BASE_DIR}/${INSTALLERLNG}"; then
+				echo "ok"
```

Comment:
Two tabs, I see...  That's rather a lot for a single level indent.

Reply:
See above.